### DISCUSSION TITLE
[深度许愿]有没有可能将deepin的易用性与Gentoo/Funtoo的灵活可定制性结合起来？

### DISCUSSION THREAD

**Discussion by wwlittewayne** [linuxdeepin/developer-center#4421] [Features Request & Ideas | 特性请求 & 头脑风暴]
Gentoo/Funtoo基于源码包发行，可定制性极强，可以发挥出电脑的极致性能，但是安装配置太麻烦，门槛太高。
一时异想天开：能不能将deepin的易用性与Gentoo/Funtoo的灵活可定制性结合起来？
比如，安装过程还是类似deepin的那种友好的GUI界面，但是允许用户选择各个软件用预编译的软件包还是源码包，同时deepin预先定制好“适配各种应用场景各种机型的Gentoo安装配置文件”，用户只需要在一个列表中做一个选择，省去繁琐的配置过程～

**Comment by Decodetalkers** [comment]
那你只要把DDE打包到 gentoo就行了，欢迎去做😆

**Reply by wwlittewayne** [reply]
直接弄Gentoo不是一般人玩的转的啊 ，所以才想到这个，其实Funtoo已经往这个方向前进了一大步，但是看安装说明还是需要nano手敲编辑配置文件，比如配置fstab。另外，每个软件都从源码编译也实在费事，最好可以按需选择源码安装还是二进制安装某软件。

**Comment by BLumia** [comment]
因为 edit flags 不是一个会长期无需用户介入的事情，你可能会需要处理不同组件之间由于你所设 flag 导致的问题。Gentoo 的易用性远远不止每个包的 flags 的差异，只提供个 flags 列表的界面是远远不够的。

以及，如果你是大部分应用希望用预先提供的版本，只有个别应用希望自己选特性构建的话，也可以考虑试试 distrobox 跑个 Gentoo，当然尽管这种方案你还是得自己手动编辑希望修改的配置。
